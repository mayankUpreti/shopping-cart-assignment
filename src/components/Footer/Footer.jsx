import React from 'react'


const Footer=()=>{
    return(<div className='footer'>
      Copyright &#169; 2016-2018 Sabka Bazar Grocery Supplies Private Limited
    </div>)
}

export default Footer