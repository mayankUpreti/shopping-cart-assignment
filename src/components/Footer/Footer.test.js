import React  from 'react'
import Footer from './Footer'
import {shallow} from 'enzyme';



it('testing a custom Footer ',()=>{
    expect(shallow(<Footer/>)).toMatchSnapshot()

})