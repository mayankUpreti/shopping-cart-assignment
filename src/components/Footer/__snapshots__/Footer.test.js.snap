// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing a custom Footer  1`] = `
<div
  className="footer"
  tabIndex={0}
>
  Copyright Â© 2016-2018 Sabka Bazar Grocery Supplies Private Limited
</div>
`;
